codegen/service/client.go

```diff
@@ -27,7 +27,9 @@ func ClientFile(_ string, service *expr.ServiceExpr) *codegen.File {
 			codegen.GoaImport(""),
 		}
 		imports = append(imports, svc.UserTypeImports...)
-		header := codegen.Header(service.Name+" client", svc.PkgName, imports)
+		value, ok := service.Meta.Last("goa:version:disable")
+		disableVersion := ok && value == "true"
+		header := codegen.Header(service.Name+" client", svc.PkgName, imports, disableVersion)
```

Comment:
I'm wondering if we wouldn't be better of passing the metadata object to `codegen.Header` and moving the logic that computes `disableVersion` there:
* This would remove the need for having this computation spread everywhere
* This would allow for other metadata based tweaks in the future when generating file headers